demo/scripts/controls/editor/Editor.tsx

```diff
@@ -92,6 +93,7 @@ export default function Editor(props: EditorProps) {
             experimentalFeatures: experimentalFeatures,
             undoSnapshotService: props.snapshotService,
             trustedHTMLHandler: trustedHTMLHandler,
+            sizeTransformer: size => size / scale,
```

Comment:
is there a validation on the scale prop to make sure it's greater than 0?
or maybe `size => (scale > 0 ? size / scale : size)`

Reply:
For demo site, I'd like to keep it simple and do not eat too much errors. If scale is 0, there must be code bug and we just let it throw